### ISSUE TITLE
AWS SDK not properly isolated during tests

### ISSUE THREAD

**Issue by kemitix** [kemitix/s3thorp#2]
During testing, the AWS SDK should not be used, there should be no communication required with AWS. This is need particularly for CI environments, where we don't want to have them hooked into an AWS account.

```
[info] net.kemitix.s3thorp.awssdk.S3ClientSuite *** ABORTED ***
[info]   org.scalatest.exceptions.NotAllowedException was thrown inside 
describe("objectHead"), construction cannot continue: 
"software.amazon.awssdk.core.exception.SdkClientException was thrown inside 
describe("when underlying client response is okay"), construction cannot continue: 
"Unable to load region from any of the providers in the chain 
software.amazon.awssdk.regions.providers.DefaultAwsRegionProviderChain@7dab476f: 
[software.amazon.awssdk.regions.providers.SystemSettingsRegionProvider@6ab5273c: 
Unable to load region from system settings. Region must be specified either via environment 
variable (AWS_REGION) or  system property (aws.region)., 
software.amazon.awssdk.regions.providers.AwsProfileRegionProvider@dc65ad3: No region 
provided in profile: default, 
software.amazon.awssdk.regions.providers.InstanceProfileRegionProvider@f0ad613: Unable 
to retrieve region information from EC2 Metadata service. Please make sure the application is 
running on EC2.]"" (S3ClientSuite.scala:14)
```

**Comment by kemitix** [comment]
I can replicate this error by removing my `~/.aws/` directory.

Then I can resolve the issue by setting the environment variable `AWS_REGION`.

Possibly, no code change needed, but might be able to add something to the `.travis.yml` file.

**Comment by kemitix** [comment]
closed by #9
